scripts/generate-constants.js

```diff
@@ -1,5 +1,3 @@
-#!/usr/bin/env node
```

Comment:
@Jikstra why doesn't it work with shebang? I got some weird error with desktop wrt to `generate-constants.js` but with `npm link` was able to install it.

Reply:
I don't know, i'm still debugging this weird generate-constants error. Somehow it fails on npm install but otherwise not. Don't really get it, especially as it doesn't fail on the ci...